### ISSUE TITLE
Repeated calls to `artifact add -f artifact.json --var` overwrite previous variable declarations 

### ISSUE THREAD

**Issue by pavelkucera** [gimlet-io/gimlet-cli#35]
Hi,

Repeated calls to `gimlet artifact add -f artifact.json --var <name>=value` overwrite variables declared by previous calls. This seems like a bug since the operation is called `add`.

Example:
```bash
gimlet artifact add -f artifact.json --var TEST1=test1
gimlet artifact add -f artifact.json --var TEST2=test2
```
Resulting `artifact.json`:
```json
  "context": {
    "TEST2": "test2"
  },
```

Gimlet version:
```
gimlet version 0.9.6
```

**Comment by laszlocph** [comment]
Bug confirmed.

Until the fix arrives, you may use the following as a workaround:
```
gimlet artifact add -f artifact.json --var TEST2=test2 --var TEST1=test1
```

**Comment by laszlocph** [comment]
Fix released in 0.9.7
